.github/workflows/run_pr_tests.yml

```diff
@@ -390,3 +390,67 @@ jobs:
         toolchain_file: "scripts/../platform/arm-linux/aarch64-toolchain.cmake"
         extra_flags: -DCA_BUILTINS_TOOLS_DIR=${{ github.workspace }}/llvm_install_native/bin
 
+  # Based on a combination of: mr-ubuntu-gcc-x86_64-clik
+  #                       and: mr-ubuntu-gcc-x86_64-clik-refsi
+  run-ubuntu-gcc-x86_64-clik-refsi:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 60
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v4.1.0
+    - name: Install Ninja
+      uses: llvm/actions/install-ninja@main
+    - run: echo WORKSPACE is $GITHUB_WORKSPACE && echo PWD is `pwd` && ls -al
+    - name: Run Clik
+      run: |
+        cmake -Bbuild_clik -GNinja -DCMAKE_INSTALL_PREFIX=install clik
+        LD_PRELOAD=/lib/x86_64-linux-gnu/libpthread.so.0 ninja -Cbuild_clik check
+    - name: Run Clik-Refsi # No initial clean - just point o/p at new dirs
+      run: |
+        cmake -Bbuild_clik_refsi -GNinja -DCMAKE_INSTALL_PREFIX=install_refsi -DCLIK_HAL_NAME=refsi -DHAL_REFSI_SOC=M1 -DCLIK_EXTERNAL_HAL_DIR=${{ github.workspace }}/examples/refsi/hal_refsi clik
+        LD_PRELOAD=/lib/x86_64-linux-gnu/libpthread.so.0 ninja -Cbuild_clik_refsi check
+
+  # Based on: mr-ubuntu-gcc-x86_64-refsi-g1-wi-cl3-0
```

Comment:
Can we add wi to a different PR please? And/Or I think at this point add it to the !pull_request flag.

Reply:
pull_request flag added.